src/Symfony/Component/Translation/Tests/TranslatorTest.php

```diff
@@ -527,7 +549,6 @@ public function getValidLocalesTests()
     {
         return [
             [''],
-            [null],
```

Comment:
this test provider is used in a lot of tests and in most of them a null locale is meant to be supported. so you can't just remove this here.